plugins/MonitorStage/MonitorMain.qml

```diff
@@ -11,7 +11,7 @@ Rectangle
 {
     id: viewportOverlay
 
-    property bool isConnected: Cura.MachineManager.activeMachineHasActiveNetworkConnection || Cura.MachineManager.activeMachineHasActiveCloudConnection
+    property bool isConnected: Cura.MachineManager.activeMachineIsUsingCloudConnection
```

Comment:
This should look at any network connection, not just cloud.

Reply:
Ah yes. True. Good catch.